bindings/python/src/OpenSpaceToolkitPhysicsPy/Unit.cpp

```diff
@@ -38,7 +38,7 @@ inline void OpenSpaceToolkitPhysicsPy_Unit(pybind11::module& aModule)
             )doc"
         )
         .value(
-            "None",
+            "NoneType",
```

Comment:
```suggestion
            "None_",
```
Would this work as well?
Unit.Type.NoneType feels a bit awkward.
We could also potentially rename this to Dimensionless?

Reply:
I like Dimensionless!